### PR TITLE
Pydocs

### PR DESCRIPTION
**Pull request by MartijnKaijser**
What should we tell devs about "duration". You can set if with "duration" but also with "streaminfo". Add a note saying either one or the other?

### CODE REVIEW

**Comment by Razzeee** [comment]
Well duration is the scraped one and streaminfo is the one read from your file. They may differ due to special editions etc

**Comment by MartijnKaijser** [comment]
Well this is python API and not scraper API ;)

**Comment by Razzeee** [comment]
Does that change what should be in the fields?

File: xbmc/interfaces/legacy/ListItem.h [hunk 1]
```diff
@@ -222,8 +222,9 @@ namespace XBMCAddon
        *     - lastplayed    : string (%Y-%m-%d %h:%m:%s = 2009-04-05 23:16:04)
        *     - album         : string (The Joshua Tree)
        *     - artist        : list (['U2'])
-       *     - votes         : string (12345 votes)
+       *     - votes         : integer (12345)
        *     - trailer       : string (/home/user/trailer.avi)
+       *     - date          : string (%Y-%m-%d %h:%m:%s = 2009-04-05 23:16:04)
```
Comment (tamland) [inline xbmc/interfaces/legacy/ListItem.h]: What does setting the "date" of an item mean/do?

File: xbmc/interfaces/legacy/ModuleXbmcplugin.h [hunk 2]
```diff
@@ -103,10 +103,59 @@ namespace XBMCAddon
...
+     *        - xbmcplugin.SORTMETHOD_TITLE_IGNORE_THE
+     *        - xbmcplugin.SORTMETHOD_TRACKNUM
+     *        - xbmcplugin.SORTMETHOD_UNSORTED
+     *        - xbmcplugin.SORTMETHOD_VIDEO_RATING
+     *        - xbmcplugin.SORTMETHOD_VIDEO_RUNTIME
+     *        - xbmcplugin.SORTMETHOD_VIDEO_SORT_TITLE
+     *        - xbmcplugin.SORTMETHOD_VIDEO_SORT_TITLE_IGNORE_THE
+     *        - xbmcplugin.SORTMETHOD_VIDEO_TITLE
+     *        - xbmcplugin.SORTMETHOD_VIDEO_USER_RATING
+     *        - xbmcplugin.SORTMETHOD_YEAR
```
Comment (tamland) [inline xbmc/interfaces/legacy/ModuleXbmcplugin.h]: didn't you already do (340856371a0f6752cd5bc04753ec2c5510d03980) and undo (336792b266bde53c59d2ba4d5fca79b0985a799c) this once?:)

File: xbmc/interfaces/legacy/ListItem.h (hunk 1 above)
Comment (ronie) [inline xbmc/interfaces/legacy/ListItem.h]: it's usually the aired date of an episode

Comment (tamland) [inline xbmc/interfaces/legacy/ListItem.h]: there's already an 'aired' property

Comment (ronie) [inline xbmc/interfaces/legacy/ListItem.h]: for episodes, both return the same value indeed.

there are various use cases of the 'date' infolabel, depending on the context:
tv shows: first aired date of a tvshow
pictures: date/time the picture was taken
file: date/time stamp of the file

**Pull request closed without merging**